applications/IgaApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -293,6 +283,12 @@ void RegisterSurfaceGeometryBase(
         .def_property_readonly("DomainV", &Type::DomainV)
         .def_property_readonly("NbKnotsU", &Type::NbKnotsU)
         .def_property_readonly("NbKnotsV", &Type::NbKnotsV)
+        .def_property_readonly("NbPolesU", &Type::NbPolesU)
```

Comment:
I just saw that you use `Number` a bit inconsistently
In other parts of Kratos we use either `NumberOf` or `Num...`